### DISCUSSION TITLE
"Your GitHub academic discount coupon has expired"

### DISCUSSION THREAD

**Discussion by csutter** [community/community#22778] [New to GitHub]
<p>Hey :wave:</p>
<p>I have the same issue someone else on here had a few days ago (<a href="https://github.community/t/your-github-academic-discount-coupon-has-expired-message-wont-go-away/135936" class="inline-onebox">"Your GitHub academic discount coupon has expired" message won't go away</a>).</p>
<p>I took advantage of the Github Student program a while back when I went back to university to do another degree - I’ve since graduated and am no longer eligible, which is fine obviously, but despite downgrading my account, the huge red banner saying “Your GitHub academic discount coupon has expired” isn’t going away. I’ve filed a support ticket months ago and bumped it last week but received no response.</p>
<p>Could you please push through my downgrade as well?</p>
<p>Thanks,<br>
Christian</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>That’s done for you now. I checked and it looked like your ticket was in the wrong queue. Sorry for the wait!</p>

**Reply by JimmyIm** [reply]
hi i have same issue for starup account  mistakenly it was coming to other email and expired 


 
--


<p style="caret-color: rgb(36, 41, 46); color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot; !important; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: -webkit-center; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; margin-top: 0px; margin-bottom: 10px;">Greetings,<span class="Apple-converted-space"> </span><strong style="font-weight: 600; box-sizing: border-box; font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot; !important;">lifejourneyAI</strong></p><p style="caret-color: rgb(36, 41, 46); color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot; !important; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: -webkit-center; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; margin-top: 0px; margin-bottom: 10px;">We’re sorry to say that your coupon for<span class="Apple-converted-space"> </span><span data-markjs="true" class="outlook-search-highlight"><span data-markjs="true" class="outlook-search-highlight">GitHub</span></span><span class="Apple-converted-space"> </span>Free has expired.<span class="Apple-converted-space"> </span></p><p style="caret-color: rgb(36, 41, 46); color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot; !important; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: -webkit-center; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; margin-top: 0px; margin-bottom: 10px;">If you’d like to renew<span class="Apple-converted-space"> </span><span data-markjs="true" class="outlook-search-highlight"><span data-markjs="true" class="outlook-search-highlight">GitHub</span></span>, please update your payment method.</p>

**Comment by rmverma** [comment]
Hello, My school photo does not have a date in the front and the Git form is refusing to accept it.
